rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I think that we have a small misunderstanding.

While conducting a hibernate update from 3 to 4 or from 4 to 5, you are perfectly right regarding "optional" and "mandatory" rules.

In an EAP migration project, updating Hibernate to a newer version is optional, as hibernate can be embedded in the application and disabled at application server-level. Considering a minimalistic migration (doing as few changes as possible), this is the preferred approach: we keep an embedded version of Hibernate in the application and do not touch the source code.